Bindings/common.i

```diff
@@ -140,6 +140,14 @@ namespace OpenSim {
         $self->setColumnLabels(columnLabels);
     }
 }
+%extend OpenSim::TimeSeriesTable_<SimTK::Vec3> {
+    TimeSeriesTable_<double> flatten() {
+        return $self->flatten();
+    }
+    TimeSeriesTable_<double> flatten(std::vector<std::string> suffixes) {
+        return $self->flatten(suffixes);
+    }
+}
```

Comment:
Is this here because SWIG does not handle `using DataTable_<double, ETY>::flatten;`?